pkg/virt-handler/dmetrics-manager/dmetrics-manager.go

```diff
@@ -83,11 +83,15 @@ func (m *DownwardMetricsManager) StopServer(vmi *v1.VirtualMachineInstance) {
 		return
 	}
 
+	m.done = true
```

Comment:
Is this intentional?

Reply:
yup, it's just for consistency

Reply:
Can you explain? This means that no new VMI will be started.

Reply:
ouch, you're right! My bad, fixed.